plugins/com.google.cloud.tools.eclipse.appengine.deploy.ui.test/META-INF/MANIFEST.MF

```diff
@@ -12,6 +12,7 @@ Require-Bundle: org.hamcrest;bundle-version="1.1.0",
 Import-Package: com.google.cloud.tools.eclipse.appengine.facets,
  com.google.cloud.tools.eclipse.test.util,
  com.google.cloud.tools.eclipse.test.util.ui,
+ com.google.cloud.tools.login,
```

Comment:
Hmm, this would seem to have been required on the host bundle (`c.g.c.t.e.ae.deploy.ui`), and thus pulled in from there?